src/core/config/ProviderSettingsManager.ts

```diff
@@ -157,6 +160,14 @@ export class ProviderSettingsManager {
 					isDirty = true
 				}
 
+				// kilocode_change start
+				if (!providerProfiles.migrations.morphApiKeyMigrated) {
+					const result = await migrateMorphApiKey(this.context, providerProfiles)
+					providerProfiles.migrations.morphApiKeyMigrated = true
+					isDirty ||= result
```

Comment:
oh cool, I didn't realize our repo had support for `||=` 👏

Reply:
any reason why it might not have it?